communitiesuk/funding-service-design-assessment
# Fs 2484 sortable fields in assessment

Jira ticket reference
https://digital.dclg.gov.uk/jira/browse/FS-2484


### Description
Added sortable functionality for columns: location & funding requested



### How to test
_If manual testing is needed, give suggested testing steps_
1) Open the assessment dashboard
2) Find the sorting filter on location or funding column.
3) Click on it & observe the table sorting accordingly.


### Screenshots of UI changes (if applicable)
![image](https://user-images.githubusercontent.com/127315890/235617611-2d0816c1-c85f-4d73-b295-8816e6abd0bc.png)
